run-uts

```diff
@@ -9,22 +9,39 @@ else
     SKIP="vendor,$SKIP"
 fi
 
+echo "Removing old coverprofiles..."
+find . -name "*.coverprofile" -type f -delete
+
+echo "Calculating packages to cover..."
+go_dirs=$(find -type f -name '*.go' | \
+	      grep -vE '/vendor/|\./proto/|.glide|/k8sfv/' | \
```

Comment:
Yes, indeed - sorry for conflating these.  I'll split these to another PR.

Yes, I guess that .proto and k8sfv are Felix-specific, so will remove those.